cli/compile/compile.go

```diff
@@ -264,6 +269,28 @@ func runCompileCommand(cmd *cobra.Command, args []string) {
 	})
 	if compileError != nil && output.OutputFormat != "json" {
 		feedback.Errorf(tr("Error during build: %v"), compileError)
+
+		// Check the error type to give the user better feedback on how
+		// to resolve it
+		var platformErr *arduino.PlatformNotFoundError
+		if errors.As(compileError, &platformErr) {
+			split := strings.Split(platformErr.Platform, ":")
+			if len(split) < 2 {
+				panic(tr("Platform ID is not correct"))
+			}
+
+			pm := commands.GetPackageManager(inst.GetId())
+			platform := pm.FindPlatform(&packagemanager.PlatformReference{
+				Package:              split[0],
+				PlatformArchitecture: split[1],
+			})
+
+			if platform != nil {
+				feedback.Errorf(tr("Try running `%s core install %s`", globals.VersionInfo.Application, platformErr.Platform))
```

Comment:
```suggestion
				feedback.Errorf(tr("Try running %s", fmt.Sprintf("`%s core install %s`", globals.VersionInfo.Application, platformErr.Platform)))
```
This will avoid the "core" and "install" being mistaken as words to translate.